### PR TITLE
 Fixed support for "additionalProperties"

### PR DESCRIPTION
**Pull request by EasyTuan**
### Checklist

* [x] **I'm adding or updating code**
  - [x] I've added and/or updated tests
* [x] **I'm adding a new feature**
  - [x] I've updated the playground with an example use of the feature

### Details

I fixed the bug of the original property 'additionalProperties' and can support the delete function to beautify the style.

image

### CODE REVIEW

File: test/ObjectField_test.js [hunk 1]
```diff
@@ -216,7 +216,7 @@ describe("ObjectField", () => {
         l => l.textContent
       );
 
-      expect(labels).eql(["baz", "bar", "qux", "foo"]);
+      expect(labels).eql([]);
```
Comment (edi9999) [inline test/ObjectField_test.js]: You're probably added a bug if you change that test

File: test/ObjectField_test.js [hunk 2]
```diff
@@ -448,28 +448,6 @@ describe("ObjectField", () => {
       expect(node.querySelectorAll(".field-string")).to.have.length.of(1);
     });
 
-    it("should render a label for the additional property key", () => {
-      const { node } = createFormComponent({
-        schema,
-        formData: { first: 1 },
```
Comment (edi9999) [inline test/ObjectField_test.js]: tests should not be deleted except if the feature has been removed completely

**Comment by aerfio** [comment]
@EasyTuan hi, what's the status? Do you plan to keep on working on this PR?

**Comment by glasserc** [comment]
Closing in favor of #1123.

**Pull request closed without merging**